code-bar-job-board-app/src/components/App/App.css

```diff
@@ -95,6 +95,11 @@ header {
  .footer-link {
      color: white;
      text-decoration: none;
+     font-weight: bold;
+ }
```

Comment:
There is a lot of css here that we don't need to write here e.g the class `text-white` will make the text right, same for adding the class `font-weight-bold`

Reply:
Great - I'll change that!